contribute/dev_setup.rst

```diff
@@ -0,0 +1,100 @@
+=================
+Development Setup
+=================
+
+This section describes how someone wanting to contribute to the OpenCMISS project should set up their *working copy* for developing OpenCMISS.
+
+.. contents::
+
+Overview
+========
+
+The OpenCMISS codebase is hosted on GitHub and therefore Git is used to track changes in the codebase.  Before you begin you will need to have a few prerequisites satisfied:
+
+#. GitHub user account (for the rest of this document we will call our user *Prasad*)
+#. Git installed
+
+This document doesn't cover the process of satisfying these prerequisites, it is left as an exercise for you (Google is your friend).  The remainder of this document assumes that these prerequisites have been met.  This document covers setup from the command line if you are using a GUI like 'GitHub for Windows' or 'Pycharm' then you will need to adjust the commands for the GUI you are using.
+
+The goal here is to get a working copy of source code, tests, and documentation onto your PC so that you can begin development.  To make this happen you will need to fork a :term:`prime OpenCMISS repository`, make a clone onto your PC, and set up the Git remotes.  In :numref:`fig_devSetup_githubRepos` you can see a pictorial representation of what we are aiming to achieve.
+
+.. _fig_devSetup_githubRepos:
+
+.. figure:: images/OpenCMISSProcesses-GitHubRepos.png
+   :align: center
+   :alt: Setup of Git repositories
+
+   Setup of repositories for development
+
+The four steps to getting set up are detailed below.
+
+Forking your own copy
+=====================
+
+Login to GitHub using your credentials and go to the OpenCMISS GitHub organisation page::
+
+   https://github.com/OpenCMISS
+
+The OpenCMISS organisation contains the git repositories for each component of OpenCMISS:
+
+   - https://github.com/OpenCMISS/iron
+   - https://github.com/OpenCMISS/zinc
+   - https://github.com/OpenCMISS/examples
+   - https://github.com/OpenCMISS/documentation
+   - https://github.com/OpenCMISS/website
+   - https://github.com/OpenCMISS/manage
+
```

Comment:
These links should appear as just 'iron', 'zinc' etc. and link to the address.
Add repo cmake_modules.
I think it would be good to present this as a table with a second column describing in a minimally brief statement what the repository contains.